Classes/OldModel/Event.php

```diff
@@ -4571,4 +4522,34 @@ public function isPublished(): bool
     {
         return !$this->hasRecordPropertyString('publication_hash');
     }
+
+    private function getAssociationAsArray(string $foreignTableName, string $mmTableName, array $foreignFields): array
```

Comment:
Please add a `string[]` PHPDoc type annotation for `$foreignFields` to help static code analysis (and human readers).